NitroxClient/Debuggers/SceneDebugger.cs

```diff
@@ -2,25 +2,40 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
+using System.Reflection;
 using System.Text;
+using NitroxClient.MonoBehaviours;
 using NitroxClient.Unity.Helper;
 using UnityEngine;
 using UnityEngine.SceneManagement;
+using UnityEngine.UI;
 
 namespace NitroxClient.Debuggers
 {
     public class SceneDebugger : BaseDebugger
     {
         private Vector2 gameObjectScrollPos;
         private Vector2 hierarchyScrollPos;
+        private Vector2 monoBehaviourScrollPos;
         private GameObject selectedObject;
+        private MonoBehaviour selectedMonoBehaviour;
         private Scene selectedScene;
+        private List<DebuggerAction> actionList = new List<DebuggerAction>();
```

Comment:
Because you've created a few functions that are special for the List<DebuggerAction> maybe wrap it into a DebuggerActions class and refactor the methods that access the list into DebuggerActions as well.

Reply:
Just GettFromField() use the list so I think it's better to give the list (because GetFromField() is now in DebuggerAction) as a property...